docs/UPGRADING.md

```diff
@@ -2,6 +2,25 @@
 
 Here you can find a list of migration guides to handle breaking changes between releases of the CLI.
 
+## Unreleased
+
+### `board details` arguments change
+
+The `board details` command now accept only the `-b` flag to specify the fqbn.
+
+`board details <fqbn>` has been deprecated in favour of `board details -b <fqbn>`.
+
+### `board attach` arguments change
+
+The `board attach` command now accept `-p` and `-b` flags to respectively select the board fqbn and the port.
```

Comment:
```suggestion
The `board attach` command now uses `-p` and `-b` flags to respectively select the board FQBN and the port.
```

I think "uses" makes it more clear that this is a hard change rather than an additional syntax.